dashboard/templates/view_file_access_within_week.html

```diff
@@ -19,33 +19,31 @@
 {% endblock %}
 {% block content %}
     <!-- week slider -->
-    <br/>
-    <br/>
-    <h2>What Students Are Reading?</h2>
+    <h4 align="center">What Students Are Reading</h4>
 
 
     <input type="text" id="slider" class="slider">
 
-
     <input type="hidden" name="slider_num" id="slider_num" />
     <div/>
-
-    Top files access by
+    <br/>
+    <h6 align="center">Files accessed by
     <select class="grade" name="grade" id="grade">
-        <option value="all" selected>All Grade</option>
+        <option value="all" selected>All Grades</option>
         <option value="A" >A</option>
         <option value="B" >B</option>
         <option value="C" >C</option>
         <option value="D" >D</option>
         <option value="F" >F</option>
         <option value="NO_GRADE" >NO GRADE</option>
-    </select> <span id="slider_label">in last 1 week: </span>
-
+    </select> <span id="slider_label"> students from week 1 to: </span></h6>
+    <br/>
     <div id="chart"></div>
 {% endblock %}
 {% block js %}
     <script src="{% static 'd3/d3.js' %}"></script>
     <script src="{% static 'd3-tip/dist/index.js' %}"></script>
     <script src="{% static 'js/rSlider.min.js' %}"></script>
     <script src="{% static 'js/file_access_within_week.js'%}"></script>
+    <script src="http://labratrevenge.com/d3-tip/javascripts/d3.tip.v0.6.3.js"></script>
```

Comment:
Can you switch this to` {% static 'd3-tip/dist/index.js' %}` (0.9.1) already there. Or does it need this old version?

Reply:
No, I can definitely use the existing version. Link removed.